modules/juce_graphics/native/juce_freetype_Fonts.cpp

```diff
@@ -353,8 +353,8 @@ class FreeTypeTypeface   : public CustomTypeface
     bool getGlyphShape (Path& destShape, const FT_Outline& outline, const float scaleX)
     {
         const float scaleY = -scaleX;
-        const short* const contours = outline.contours;
-        const char* const tags = outline.tags;
+        short unsigned int* const contours = outline.contours;
+        unsigned char* const tags = outline.tags;

```

Comment:
Later versions of JUCE [changed these declarations to `auto*`](https://github.com/juce-framework/JUCE/blob/dee78f29f60dbe46e436872971b18e260522e8a4/modules/juce_graphics/native/juce_freetype_Fonts.cpp#L350-L351). Maybe that would be a more appropriate fix?